websites/B/BladeFM/dist/metadata.json

```diff
@@ -33,15 +33,15 @@
       "title": "Change Line #1 (Song Details)",
       "icon": "fas fa-music",
       "value": "Streaming to %listeners% listeners",
-      "placeholder": "%listeners%"
+      "placeholder": "%artist%, %listeners% %songText% %title%"
     },
     {
       "id": "state",
       "title": "Change Line #2 (Presenter Details)",
       "icon": "fas fa-microphone",
       "value": "%songText%",
-      "placeholder": "%artist% - %song%"
+      "placeholder": "%artist%, %listeners% %songText% %title%"
```

Comment:
maybe add a comma after each not just the first?

Reply:
shh totally didnt forget that